charts/smarter-device-manager/templates/common.yaml

```diff
@@ -0,0 +1,77 @@
+{{/* Make sure all variables are set properly */}}
+{{- include "common.values.setup" . }}
+
+{{/* Host paths */}}
+{{- define "sdm.dev.volume" -}}
+name: host-dev
+hostPath:
+  path: {{ .Values.devPath | default "/dev" }}
+{{- end -}}
+{{- define "sdm.sys.volume" -}}
+name: host-sys
+hostPath:
+  path: {{ .Values.sysPath | default "/sys" }}
+{{- end -}}
+{{- define "sdm.device-plugins.volume" -}}
+name: host-device-plugins
+hostPath:
+  path: {{ .Values.devicePluginPath | default "/var/lib/kubelet/device-plugins" }}
+{{- end -}}
+
+{{- define "sdm.dev.volumeMount" -}}
+name: host-dev
+mountPath: /dev
+{{- end -}}
+{{- define "sdm.sys.volumeMount" -}}
+name: host-sys
+mountPath: /sys
+{{- end -}}
+{{- define "sdm.device-plugins.volumeMount" -}}
+name: host-device-plugins
+mountPath: /var/lib/kubelet/device-plugins
+{{- end -}}
```

Comment:
All these should be easily doable with just common chart and values.yaml.
Why are they here?

Reply:
This is also using the abilities of the `common` library, except setting the required paths in the chart instead of the values.yaml, preventing accidental removal from the additionalVolumes list.

Reply:
Because values.yaml is for configuration. 
And that thing is rigid, not something to alter.